src/chocolatey.resources/helpers/functions/UnInstall-ChocolateyZipPackage.ps1

```diff
@@ -50,7 +50,7 @@ param(
     $zipContentFile
     $zipContents=get-content $zipContentFile
     foreach ($fileInZip in $zipContents) {
-      remove-item "$fileInZip" -ErrorAction SilentlyContinue
+      remove-item -Path "$fileInZip" -ErrorAction SilentlyContinue -Recurse -Force

```

Comment:
I would write this block with a pipeline instead of a for-loop

```powershell
Get-Content $zipContentFile | Remove-Item -Force -Recurse -ErrorAction SilentlyContinue
```